mergify_engine/worker.py

```diff
@@ -323,6 +324,17 @@ class StreamProcessor:
     redis_cache: utils.RedisCache
     dedicated: bool
 
+    owners_mapping: typing.Dict[
+        github_types.GitHubAccountIdType, github_types.GitHubLogin
+    ] = dataclasses.field(default_factory=dict)
```

Comment:
just wondering if this is not going to explode in memory size? (at least in theory)
should we leverage Redis?

Reply:
I hope to see it explode!

I don't think we should add redis yet for this.

This is just a dict of int/string of Mergify installation, we currently have 4000 installations, so if all org names are ~ 50 chars long, the dict size will be 4000 * 32 (int) +  4000 * 100 (char) + 10% (dict overhead), so less than 1Mo.

I think we can afford this in-memory cache. I can put a comment just in case to not forget it.